### PR TITLE
Update Envoy docset with versions (1.18-1.21)

### PR DESCRIPTION
**Pull request by ruoshan**
is it ok to submit multiple versions in one shot?

### CODE REVIEW

File: docsets/Envoy/docset.json [hunk 1]
```diff
@@ -1,12 +1,28 @@
...
+            "version": "1.20.1",
+            "archive": "versions/1.20.1/Envoy.tgz"
+        },
+        {
+            "version": "1.19.1",
+            "archive": "versions/1.19.1/Envoy.tgz"
+        },
+        {
+            "version": "1.18.4",
+            "archive": "versions/1.18.4/Envoy.tgz"
```
Comment (Kapeli) [inline docsets/Envoy/docset.json:24]: The archive at this path does not exist. You added `versions/v1.18.4/Envoy.tgz` instead.

File: docsets/Envoy/docset.json [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+            "version": "1.21.1",
+            "archive": "versions/1.21.1/Envoy.tgz"
+        },
+        {
+            "version": "1.20.1",
+            "archive": "versions/1.20.1/Envoy.tgz"
+        },
+        {
+            "version": "1.19.1",
+            "archive": "versions/1.19.1/Envoy.tgz"
```
Comment (Kapeli) [inline docsets/Envoy/docset.json:20]: Same as for 1.18.4

File: docsets/Envoy/docset.json [hunk 3]
```diff
@@ -1,12 +1,28 @@
...
+    "version": "1.21.1",
     "archive": "Envoy.tgz",
     "author": {
         "name": "Ruoshan Huang",
         "link": "https://twitter.com/_ruoshan"
     },
     "specific_versions": [
+        {
+            "version": "1.21.1",
+            "archive": "versions/1.21.1/Envoy.tgz"
```
Comment (Kapeli) [inline docsets/Envoy/docset.json:12]: Same as for 1.18.4

**Comment by ruoshan** [comment]
@Kapeli PTAL again. just fixed the directories.

**Comment by Kapeli** [comment]
Thank you! Sorry for the noise from the github-actions bot, I'm trying to set up a docset verification script that comments the results of the verification in the pull request.

Yes, it's ok to submit multiple versions in a pull request. You can also submit updates for multiple docsets within a pull request.

**Pull request merged**